### PR TITLE
Frogerme menus cli

### PR DESCRIPTION
**Pull request by froger-me**
Add support for listing, creating and deleting Official Account menus.

### CODE REVIEW

File: src/Commands/Menus/CreateMenus.php [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+        $app = Factory::officialAccount([
+            'app_id' => $appId,
+            'secret' => $secret,
+            'token' => $token,
+            'aes_key' => $aesKey,
+            'log' => [
+                'level'      => 'debug',
+                'permission' => 0777,
+                'file'       => __DIR__ . '/../../../../../../tmp/easywechat.log',
```
Comment (overtrue) [inline src/Commands/Menus/CreateMenus.php]: ？`/tmp/easywechat/easywechat.log`

File: src/Commands/Menus/DeleteMenus.php [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+use EasyWeChat\Console\Commands\Command;
+use EasyWeChat\Factory;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+/**
+ * Class RsaPublicKey
```
Comment (overtrue) [inline src/Commands/Menus/DeleteMenus.php]: class name

File: src/Commands/Menus/DeleteMenus.php [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+        $app = Factory::officialAccount([
+            'app_id' => $appId,
+            'secret' => $secret,
+            'token' => $token,
+            'aes_key' => $aesKey,
+            'log' => [
+                'level'      => 'debug',
+                'permission' => 0777,
+                'file'       => __DIR__ . '/../../../../../../tmp/easywechat.log',
```
Comment (overtrue) [inline src/Commands/Menus/DeleteMenus.php]: log path

File: src/Commands/Menus/ListMenus.php [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+        $app = Factory::officialAccount([
+            'app_id' => $appId,
+            'secret' => $secret,
+            'token' => $token,
+            'aes_key' => $aesKey,
+            'log' => [
+                'level'      => 'debug',
+                'permission' => 0777,
+                'file'       => __DIR__ . '/../../../../../../tmp/easywechat.log',
```
Comment (overtrue) [inline src/Commands/Menus/ListMenus.php]: +

**Review by froger-me — COMMENTED** [review]
Thank you for the feedback and for still considering the pull request given the silly omissions - I guess I was too eager and didn't pay attention.
Changes performed.

**Comment by overtrue** [comment]
@froger-me Thank you bro.

**Pull request merged**